### ISSUE TITLE
Filter by book type

### ISSUE THREAD

**Issue by getBoolean** [ACK72/kavya-aidoku#1]
Currently it is showing light novels and manga, but Aidoku doesnt support light novels. Is there a way to filter to only include manga?

A workaround is to create a _Kavita_ user and only give it access to the manga libraries, and use the API key from that user in Aidoku

**Comment by ACK72** [comment]
I could implement an option to filter and display those libraries,
but there are currently no slots left in the aidoku testflight for me to test them.

I know there is a way to do it via sideloading, but it's quite cumbersome and I'm not considering it at the moment.
If you want a quick update for it, please leave a reply.

**Comment by getBoolean** [comment]
yeah I can test it for you, but this isnt a major issue since the workaround works pretty well

**Comment by LordFransie** [comment]
I would love to help with this.
